src/test/java/com/lightstep/tracer/shared/OptionsTest.java

```diff
@@ -117,10 +117,6 @@ public void testOptionsBuilder_noComponentName() throws Exception {
         Object componentName = options.tags.get(COMPONENT_NAME_KEY);
         assertNotNull(componentName);
 
-        // the actual value will vary depending on how the tests are run, but should always
-        // contain 'Main'
-        assertTrue("Unexpected sun.java.command value", componentName.toString().contains("Main"));
```

Comment:
This was a not very valuable assertion, as stated, the actual value would depend on how the tests are run. Now that we're running them with Maven, this assertion is failing and I was unable to find a common string to look for in the value.